lua/cmp/autocmds.lua

```diff
@@ -1,23 +1,22 @@
 local M = {}
-
-local v = vim.version()
-local has_native_autocmd = v.major >= 0 and v.minor >= 7
 local emit = require('cmp.utils.autocmd').emit
 
-local function nvim_auto()
+function M.autocmd()
   local ___cmp___ = vim.api.nvim_create_augroup('___cmp___', { clear = true })
   vim.api.nvim_create_autocmd('InsertEnter', {
     group = ___cmp___,
     callback = function()
       emit('InsertEnter')
     end,
+    desc = [[autocmd InsertEnter * lua require'cmp.utils.autocmd'.emit('InsertEnter')]],
```

Comment:
I would recommend making those descriptions human readable rather than insinuating they look like viml version.

`desc = 'nvim-cmp: emits InsertEnter event(?)',`